docs/kubernetes/ingress-usage.md

```diff
@@ -240,6 +240,26 @@ a unique value. Read more about this in our
 
      flowId("reuse")
 
+# Filters - return fast
+
+Sometimes you just want to return a header, redirect or even static
+html content. You can return from skipper without doing a proxy call
+to a backend, if you end your filter chain with `<shunt>`.
+
+## Return static content
+
+To set a response header `X: bar`, return
+`<html><body>hello</body></html>` in the HTTP response body and return
+HTTP status code 200 from ingress itself, you can use the following
```

Comment:
"from the ingress itself". This doesn't seem to be specific to skipper's usage as an ingress. Should we rather promote a general documentation entry for the features that work as well when used as a simple proxy and reference them from the Kubernetes ingress specific entries?

Reply:
@lmineiro I don't think it makes sense, because it is right now mostly used as ingress.
I am open to get PRs that refactor the docs, but I do not think it is a good way to split these. In the end we have godoc for all and basically only a few documentation for other dataclients. If you like to provide more docs, feel free to provide them. I don't agree spreading the docs for users makes sense.